test/unit/conditions/context.test.ts

```diff
@@ -90,20 +90,16 @@ describe('context parameters', () => {
     };
     const condition = new ContractCondition(conditionObj);
     const conditionExpr = new ConditionExpression(condition);
-    const conditionContext = conditionExpr.buildContext(provider, {}, signer);
     expect(conditionExpr.contextRequiresSigner()).toBe(true);
-    expect(conditionContext.requiresSigner()).toBe(true);
   });
 
   it('detects if a signer is not required', () => {
     const condition = new RpcCondition(testRpcConditionObj);
     const conditionExpr = new ConditionExpression(condition);
-    const conditionContext = conditionExpr.buildContext(provider, {}, signer);
     expect(JSON.stringify(condition.toObj()).includes(USER_ADDRESS_PARAM)).toBe(
       false
     );
     expect(conditionExpr.contextRequiresSigner()).toBe(false);
```

Comment:
Similar comment as above, for completeness:
```suggestion
    expect(conditionExpr.contextRequiresSigner()).toBe(false);

    // check that conditionContext created successfully even if signer passed
    expect(conditionExpr.buildContext(provider, {}, signer)...to succeed
    
    // try to create condition context without signer
    expect(conditionExpr.buildContext(provider, {})....to succeed
```